server/src/main/scala/services/graphstages/ArrivalsGraphStage.scala

```diff
@@ -23,6 +23,23 @@ case object ForecastArrivals extends ArrivalsSourceType
 
 case object BaseArrivals extends ArrivalsSourceType
 
+object ArrivalsGraphStage {
+  def terminalRemovals(incomingArrivals: Iterable[Arrival], existingArrivals: Iterable[Arrival]): Iterable[Arrival] =
+    existingArrivals.filter { oldArrival =>
+      val oldKey = oldArrival.unique
+      val noLongerExists = !incomingArrivals.exists(_.unique == oldKey)
```

Comment:
as it boolean may be isNoLongerExists would be better name ?

Reply:
`noLongerExists` reads well as a boolean, ie `if (noLongerExists) x else y`